core-services/prow/02_config/_config.yaml

```diff
@@ -179,14 +179,23 @@ plank:
         entrypoint: gcr.io/k8s-prow/entrypoint:v20210730-987d7118e9
         initupload: gcr.io/k8s-prow/initupload:v20210730-987d7118e9
         sidecar: gcr.io/k8s-prow/sidecar:v20210730-987d7118e9
-    openshift-priv:
+  - cluster: arm[0-9]{2}
```

Comment:
Are you sure this supports regex?

Reply:
yes, at least that's what the README says: https://github.com/kubernetes/test-infra/tree/master/prow/plank

Reply:
I think the readme is wrong: https://github.com/kubernetes/test-infra/blob/8f639d55cbd9aeeaabb4e8f35042b6b9f94d1dd6/prow/config/config.go#L567-L572

Reply:
I've changed the cluster value to hardcode `arm01` for now, and also opened https://github.com/kubernetes/test-infra/issues/23096